content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,132 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
 
-<TopBlock>
+## Overview
 
-Platform Console enables you to integrate the latest Prisma Data Platform products into your application
+The [Platform Console](http://console.prisma.io) enables you to manage and configure your projects that use Prisma Data Platform (PDP) products, and helps you integrate them into your application:
 
 - [Accelerate](/accelerate/what-is-accelerate): Global database cache with scalable connection pooling
 - [Pulse](/pulse/what-is-pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
 
-[Go to Platform Console](https://console.prisma.io)
 
-</TopBlock>
+## Concepts
+
+The Platform Console workflows are based on four main concepts:
+
+- [**User account**](#user-account): In order to use PDP products, you need to have a PDP user account. A _user_ will typically create one user account to manage all their workspaces, projects and environments. The _user_ can also be invited to join other workspaces to collaborate on the projects in that workspace.
+- [**Workspaces**](#workspace): A user account can belong to multiple workspaces. A workspace typically represents a _team_ of individuals working together on one or more projects. **Billing is on a workspace level**, i.e. the invoice for a workspace at the end of the month captures all costs associated with the projects in a given workspace.
+- [**Project**](#project): A project belongs to a workspace. It typically represents the _application_ or _service_ a team is working on.
+- [**Environments**](#environment): An environment belongs to a project. It typically maps to a _development stage_, like `dev`, `staging` or `prod`. **API keys are provisioned on the environment level**, and products are configured per environment as well (e.g. the database connection string used for Accelerate).
+
+Here is a visual illustration of how these concepts relate to each other:
+
+![](/img/platform/pdp-concepts.png)
+
+### User account
+
+A user account is the prerequisite for any interactions with PDP products. You can use it to manage your workspaces (and their projects). A user account can be invited to collaborate on workspaces created by other users as well.
+
+If you need to delete your user account, go [here](/platform/support#deleting-your-pdp-account).
+
+### Workspace
+
+You can create several workspaces. A workspace is an isolated space to host projects. A workspace can have multiple user accounts associated with it so that multiple users can collaborate on the the projects in the workspace.
+
+In each workspace, you can:
+
+- view and manage all projects (and their environments) in that workspace.
+- manage billing, i.e. select a [pricing plan](https://prisma.io/pricing?utm_source=prisma-docs&utm_medium=docs-site), configure payment methods, or view the invoice history.
+- view the usage of your enabled PDP products across all projects in that workspace.
+- manage team members and change their roles / permissions in the workspace.
```

Comment:
Do we actually have roles, permissions right now?

Reply:
Actually no, good catch! 

<img width="506" alt="image" src="https://github.com/prisma/docs/assets/4058327/1ccc7eda-18e0-49ad-a7d6-9c80762cac02">

Do we want to still mention this with a "coming soon" maybe? Or drop altogether?


```suggestion
- (coming soon) manage team members and change their roles / permissions in the workspace.
```

Reply:
Sounds good! But we should mention that other users can be invited to collaborate, right?

```suggestion
- invite other users to collaborate on the workspace.
```